### PR TITLE
[WIP] During CGMES import, correct nominal voltage if possible while creating fictitious voltage levels for line containers

### PR DESCRIPTION
**Pull request by miovd**
Signed-off-by: VEDELAGO MIORA <miora.vede@gmail.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Bug fix

**What is the current behavior?** *(You can also link to an open issue here)*
Depending on the order of creation of line containers, nominal voltage can be badly set.

**What is the new behavior (if this is a feature change)?**
From the configuration of line containers we have, this solution allows to only have correct nominal voltages.

### CODE REVIEW

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java [hunk 1]
```diff
@@ -460,8 +470,8 @@ private VoltageLevel newFictitiousVoltageLevel(Context context, LineContainerFic
                         context.nodeBreaker()
                                 ? TopologyKind.NODE_BREAKER
                                 : TopologyKind.BUS_BREAKER)
-                .setLowVoltageLimit(vlref != null ? vlref.getLowVoltageLimit() : 1.0)
```
Comment (annetill) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java:463]: I am not sure that it is correct to set the nominal voltage to 1 here too... how do you retrieve the reference voltage level for a client connection in a middle of a line?

Comment (miovd) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java:463]: We can't, this is why it is done a posteriori. I checked for every line configuration we have, they end up being set even if not done immediately.

**Review by marcosmc — COMMENTED** [review]
We have reviewed the changes and they are correct, but it is somewhat confusing to understand their purpose. 
We would like to analyze and make a proposal that takes into account all potential configurations (where fictitious voltage levels do not have adjacent real voltage levels) that may arise without excessively penalizing the conversion performance.

**Comment by miovd** [comment]
> We have reviewed the changes and they are correct, but it is somewhat confusing to understand their purpose. We would like to analyze and make a proposal that takes into account all potential configurations (where fictitious voltage levels do not have adjacent real voltage levels) that may arise without excessively penalizing the conversion performance.

The issue was when a AC line between two fictitious voltages was at the beginning of the list... However, we might change our proposition as well. I look forwards to your insights!

**Comment by marcosmc** [comment]
> From the configuration of line containers we have, this solution allows to only have correct nominal voltages.

In draft # 1905 we have developed our proposal to solve the bug in the fictitious voltage levels, our proposal consists of locating for each line container a Voltage level that serves as a reference to create all the necessary fictitious ones. We have done a performance analysis and this is affected by a slight increase in conversion, about an extra 1%

**Comment by miovd** [comment]
Replaced by #1905

**Pull request closed without merging**